Cargo.nix

```diff
@@ -1,22 +1,31 @@
+let
+  profilesByName = {
+  };
+in
```

Comment:
Minor nitpick, but does this need to be defined in a separate let-in block? I think it could also go in the let-in block beginning on lines 17-18, and it would look a tad nicer.

Reply:
I move this outside of the returned function to avoid allocating the profiles every time that function is called. But on second thought, that function is only called twice, so yeah, I'll just move it back.